channels/auth.py

```diff
@@ -24,6 +23,9 @@ def get_user(scope):
     Return the user model instance associated with the given scope.
     If no user is retrieved, return an instance of `AnonymousUser`.
     """
+    # postpone model import to avoid ImproperlyConfigured error before ASGI app is ready
```

Comment:
`...before Django setup is complete`? Maybe.

Reply:
done